canary/producer-c/jobs/orchestrator.groovy

```diff
@@ -115,7 +115,7 @@ pipeline {
                             COMMON_PARAMS << string(name: 'GIT_HASH', value: gitHash)
                         }
 
-                        // TODO: Use matrix to provide configurability in parameters
+                        TODO: Use matrix to provide configurability in parameters
```

Comment:
How do the comments work in this script?

Reply:
My bad. The `//` got removed when I was uncommenting something. Fixed it.